__tests__/index.test.js

```diff
@@ -80,6 +84,84 @@ describe("2. GET /api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/9999")
       .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toEqual("article not found");
+      });
+  });
+});
+
+//incrementing votes values
+describe("3. PATCH /api/articles/:article_id", () => {
+  test("status:200, responds with the incremented votes", () => {
+    const incrementedVotes = { inc_votes: 12 };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(incrementedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(Object.keys(articles).length).toBe(7);
+        expect(articles).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 12,
+        });
+      });
+  });
+
+  test("status:200, responds with the incremented votes", () => {
+    const incrementedVotes = { inc_votes: -100 };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(incrementedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(Object.keys(articles).length).toBe(7);
+        expect(articles).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: -100,
+        });
+      });
+  });
+
+  test("GET:400 responds with an appropriate error message when provided an incorrect request to increment votes)", () => {
+    const incrementedVotes = { inc_votes: "bananas" };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(incrementedVotes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad Request");
+      });
+  });
+
+  test("GET:400 response when no incrementVotes is added to the request body", () => {
+    const incrementedVotes = {};
+    return request(app)
+      .patch("/api/articles/3")
+      .send(incrementedVotes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toEqual("Missing key");
+      });
+  });
+
+  test("GET:404 when submitted an incorrect path", () => {
+    const incrementedVotes = {};
+    return request(app)
+      .patch("/api/artifacts/3")
+      .send(incrementedVotes)
+      .expect(404)
       .then((response) => {
         expect(response.body.msg).toEqual("Route not found");
       });
```

Comment:
This test isn't sepcific to this endpoint as it is effectively testing /api/anythingthatisntvalid, maybe put it in it's own describe block